purchases/src/main/kotlin/com/revenuecat/purchases/deeplinks/DeepLinkParser.kt

```diff
@@ -0,0 +1,28 @@
+package com.revenuecat.purchases.deeplinks
+
+import android.net.Uri
+import com.revenuecat.purchases.ExperimentalPreviewRevenueCatPurchasesAPI
+import com.revenuecat.purchases.Purchases
+import com.revenuecat.purchases.common.debugLog
+
+internal class DeepLinkParser {
```

Comment:
Maybe this can be an object, or will it have instance state later? (Or even a global internal extension function if you like.)